api/v1alpha1/jwt_types.go

```diff
@@ -81,3 +87,9 @@ type ClaimToHeader struct {
 	// to separate the JSON name path.
 	Claim string `json:"claim"`
 }
+
+// JWTExtractor defines a custom JWT token extraction from HTTP request.
```

Comment:
we need to add a doc string here mentioning expected behavior when multiple values are set i.e. multiple cookies are set

Reply:
added